src/www/services_checkip.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+/*

```

Comment:
No Copyrights for you?

Reply:
Only thing I'm interested in is retaining my right to use personally or in other projects.
As far as anyone else using it.  Far as I'm concerned my portions of the code is public domain.
If necessary for that I guess you could add a NOYB at Github copyright.